src/SapiStatsDispatcher.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Drupal\sapi;
+
+use Drupal\Core\Config\ConfigManager;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\Core\Routing\CurrentRouteMatch;
+use \Drupal\sapi\StatisticsItemInterface;
+
+/**
+ * Class SapiStatsDispatcher.
+ *
+ * @package Drupal\sapi
+ */
+class SapiStatsDispatcher implements SapiStatsDispatcherInterface {
+
+  /**
+   * Drupal\Core\Config\ConfigManager definition.
+   *
+   * @var Drupal\Core\Config\ConfigManager
+   */
+  protected $config_manager;
```

Comment:
Class properties should be lowerCamel cased (https://www.drupal.org/node/608152#naming).